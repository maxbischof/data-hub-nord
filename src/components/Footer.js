import React from 'react'
import styled from "styled-components"
import {ReactComponent as TwitterIcon} from '../icons/twitter.svg'
import {ReactComponent as LinkedInIcon} from '../icons/linkedin.svg'

export default function Footer() {

  return <StyledFooter>
      <a href="https://twitter.com/datahubnord">
        <TwitterIcon />
        <span>Twitter</span>
      </a>
    
      <a href="https://www.linkedin.com/in/maxbischof/">
        <LinkedInIcon />
        <span>LinkedIn</span>
      </a>
  </StyledFooter>
}

const StyledFooter = styled.footer`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 60px;
  border-top: 1px solid var(--grey);
  margin-top: 30px;

  span {
    font-size: 12px;
    margin-top: 6px;
  }

  a {
    display: flex;
    align-items: center;
    flex-direction: column;
    padding: 0 15px 0 15px;
    text-decoration: none;
  }

  a:visited {
    color: var(--grey);
  }
`
