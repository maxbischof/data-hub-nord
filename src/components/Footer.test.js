import React from 'react'
import Footer from './Footer'
import renderer from 'react-test-renderer'

it('renders footer component', () => {
  const renderedFooter = renderer.create(<Footer />).toJSON()
  expect(renderedFooter).toMatchSnapshot()
})
