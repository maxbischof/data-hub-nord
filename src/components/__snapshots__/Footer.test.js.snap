// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component 1`] = `
<footer
  className="sc-AxjAm hZHEwo"
>
  <a
    href="https://twitter.com/datahubnord"
  >
    <svg>
      twitter.svg
    </svg>
    <span>
      Twitter
    </span>
  </a>
  <a
    href="https://www.linkedin.com/in/maxbischof/"
  >
    <svg>
      linkedin.svg
    </svg>
    <span>
      LinkedIn
    </span>
  </a>
</footer>
`;
